src/FakerDotNet/Faker.cs

```diff
@@ -15,5 +15,6 @@ public static class Faker
         public static INumberFaker Number { get; } = Container.Number;
         public static IRandomFaker Random { get; } = Container.Random;
         public static ITimeFaker Time { get; } = Container.Time;
+        public static IBeerFaker Beer { get; } = Container.Beer;
```

Comment:
These properties should be in alphabetical order (as per the contents on the README).